RIOT-OS/RIOT
# uncrustify: replace deprecated `nl_func_var_def_blk` option

### Contribution description

The `nl_func_var_def_blk` option is depracated, it has been replaced by `nl_var_def_blk_end_func_top`. This applies the change to our configuration file.

### Testing procedure
Run uncrustify on some file using RIOT config file. Before it would display:
```
uncrustify-riot.cfg:34: option 'nl_func_var_def_blk' is deprecated; it has been replaced by 'nl_var_def_blk_end_func_top'.
You can also use 'nl_var_def_blk_end' for additional functionality
```

Now the warning should be gone.

### Issues/PRs references
None
